contracts/protocol/modules/v1/AmmModule.sol

```diff
@@ -419,6 +419,12 @@ contract AmmModule is ModuleBase, ReentrancyGuard {
 
         // Loop through and approve total notional tokens to spender
         for (uint256 i = 0; i < _actionInfo.components.length ; i++) {
+            _actionInfo.setToken.invokeApprove(
```

Comment:
I'd nit by saying adding a comment above this to explain the intent behind these lines.

Reply:
added here [52462f7](https://github.com/IndexCoop/index-protocol/pull/29/commits/52462f76cd463bf18dbb80d7831c0953e87273e0)